lib/ansible/module_utils/k8s/common.py

```diff
@@ -246,7 +246,7 @@ def load_resource_definitions(self, src):
             self.fail(msg="Error accessing {0}. Does the file exist?".format(path))
         try:
             with open(path, 'r') as f:
-                result = list(yaml.safe_load_all(f))
+                result = list(filter(None, yaml.safe_load_all(f)))
```

Comment:
my understanding is that `filter` is less pythonic than list comprehensions:

`result = [item for item in yaml.safe_load_all(f) if item]`

I also struggled with the readability of the `filter` test case - I would naively expect that to return all elements that were `None`

Reply:
I agree, new commit has been added.